test-framework/test-suites/integration/tests/list/test_list_host_profile.py

```diff
@@ -2,6 +2,11 @@
 
 
 def test_list_host_profile(host, add_host_with_net, revert_export_stack_carts, revert_etc):
+	# Add an attr with values XML doesn't like
+	result = host.run('stack set host attr backend-0-0 attr=test value=foo\\&\\<\\>\\\'\\\"')
```

Comment:
I think this might be clearer if it were a separate test.

Reply:
I made it a separate test.